### PR TITLE
[4.0] administrator login style change

### PR DESCRIPTION
**Pull request by travisrisner**
### Summary of Changes
Small style adjustment to login module for administrator. Swapped the password label to use a Font Awesome icon and forced them to the same width. This makes the fields line up.

### Testing Instructions
Check that the labels and inputs line up across all viewports.

### Expected result
The labels and input fields line up appropriately.

### Documentation Changes Required
No documentation changes I can think of.

### CODE REVIEW

File: administrator/modules/mod_login/tmpl/default.php [hunk 1]
```diff
@@ -53,7 +53,9 @@ class="form-control input-full"
 			<div class="input-group">
 				<span class="input-group-prepend">
 					<span class="sr-only"><?php echo Text::_('JSHOW'); ?></span>
-					<span class="input-group-text icon-eye" aria-hidden="true"></span>
+					<span class="input-group-text" aria-hidden="true">
```
Comment (Quy) [inline administrator/modules/mod_login/tmpl/default.php]: `aria-hidden="true"` should be moved to the icon span.

Comment (travisrisner) [reply administrator/modules/mod_login/tmpl/default.php]: Good catch!

**Comment by infograf768** [comment]
After entering the password and therefore getting the eye icon, input does not line up anymore.
screen shot 2019-01-15 at 09 11 54

**Comment by brianteeman** [comment]
@infograf768 that's possibly the js coming into effect - I said it was wrong to have the js created that way :(
Anyway not really any point atv all in this PR as the template design is taking place behind closed doors somewhere else

**Comment by travisrisner** [comment]
Fair enough. That is good to know!

**Pull request closed without merging**